platform/bind/mongo-db/index.js

```diff
@@ -0,0 +1,31 @@
+module.exports.platform = {
+  config : {
+    nodes : {
+      native : [
+        'insert',
+        'update',
+        'get',
+        'delete',
+        'search/search',
+        'search/resolve',
+        'search/filter',
+        'search/sort',
+        'search/limit',
+        'search/offset',
+      ]
+    },
+    aliases: {
+    //   '/db/insert': '/firestore/insert',
```

Comment:
is there a particular reason these aliases are commented? I think they should be uncommented and changed to 'mongo-db/*'. other than that, it seems pretty nice, great job 👍

Reply:
Thanks for the complement. I added aliases. Please pull the branch.